### DISCUSSION TITLE
different overlapping regions are identified as differentially accessible for the same  gene

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#782] [Q&A] [answered]
hI Tim,
I have identified the differentially accessible peaks for each cell type following pipeline https://satijalab.org/signac/articles/pbmc_vignette.html 
The only difference is that I used `FindAllMarkers` function instead of  `FindMarkers` demonstrated in the pipeline.

As a result, I found two differentially accessible  peaks that are for CD86  in monocytes: 

```
open_mono_all <- Mono_DE[Mono_DE$avg_log2FC > 0.5,"gene"]
closest_genes_mono_all <- ClosestFeature(integrated.37, regions = open_mono_all)
CD86.mono <- subset(closest_genes_mono_all, gene_name == "CD86")# 
#closest_region           query_region distance
#ENSSSCG00000011877   13-138427532-138504306 13-138488100-138490487        0
#ENSSSCG00000011877.1 13-138427532-138504306 13-138454303-138456247        0
```
In the code above, `integrated.37` is the integrated scATAC dataset whose peaks are created by reducing the peaks of two samples by using `reduce` function.
I understand that the rownames are ENSSSCG00000011877  and ENSSSCG00000011877.1 since they are created by comparing monocyte to different other cell types. Their closest region are the same. But I do not understand why I have two different and non-overlapping query region here ( 13-138488100-138490487 and 13-138454303-138456247 ). Are they both part of the same peak? Or actually they are two independent peaks? 

Thanks
Best
Penny

**Comment by timoast** [comment, marked as answer]
These are two different peaks that were both identified as differentially accessible and both overlap the CD86 gene. You can use the `CoveragePlot()` function to visualize the region.
